src/rules/__tests__/no-done-callback.test.ts

```diff
@@ -89,7 +107,7 @@ ruleTester.run('no-done-callback', rule, {
               messageId: 'suggestWrappingInPromise',
               data: { callback: 'done' },
               output:
-                'test("something", () => {return new Promise((done) => {done();})})',
+                'test("something", () => {return new Promise(done => {done();})})',
```

Comment:
it should probably preserve the parens if they were there originally?

Reply:
Best practice for fixers are to avoid doing stylistic fixes since that should be enforced by another rule (which will be automatically fixed if possible), to help keep fixes as small as possible.